src/Simplify.Web/Model/Validation/Attributes/MaxAttribute.cs

```diff
@@ -36,22 +87,32 @@ public override void Validate(object? value, PropertyInfo propertyInfo, IDIResol
 	{
 		if (value == null)
 			return;
-		
-		if (value is not IComparable comparableValue)
-			throw new ArgumentException($"The type of specified property value must be inherited from {typeof(IComparable)}");
 
-		ValidateTypesMatching(comparableValue);
-		
+		var maxValue = ConvertToIComparable(MaxValue);
+		var comparableValue = (IComparable)value;
```

Comment:
if `value` is not `IComparable` you will get cast exception. Need to check it for `IComparable` and add a test for that case.

Reply:
I think I won’t be able to test this because the constructor does not accept values ​​except string, int, double

Reply:
You have "public MaxAttribute(Type type, string maxValue"
So you can pass wrong type for test:
```
[Min(typeof(object), "12.5")]
public object Param { get; set; }
```

Check can be done in constructor.

Reply:
Ok. what kind of exception i need to throw?

Reply:
`ArgumentException`.
"type is not IComparable, Actual type: {type.Name}"